code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -878,3 +878,46 @@
 				user.reagents.add_reagent("syndicate_nanites", 15)
 	else
 		processing_objects.Remove(src)
+
+//Syndicate Chaplain Robe (WOLOLO!)
+/obj/item/clothing/suit/hooded/chaplain_hoodie/missionary_robe
+	description_antag = "This robe is made of reinforced fibers, granting it superior protection. The robes also wirelessly generate power for the neurotransmitter in the linked missionary staff while being worn."
+	armor = list(melee = 10, bullet = 10, laser = 5, energy = 5, bomb = 0, bio = 0, rad = 15)
+	var/obj/item/weapon/nullrod/missionary_staff/linked_staff = null
+
+/obj/item/clothing/suit/hooded/chaplain_hoodie/missionary_robe/Destroy()
+	if(linked_staff)	//delink on destruction
+		linked_staff.robes = null
+		linked_staff = null
+	..()
+
+/obj/item/clothing/suit/hooded/chaplain_hoodie/missionary_robe/equipped(mob/living/carbon/human/H, slot)
+	if(!istype(H) || slot != slot_wear_suit)
+		if(src in processing_objects)
+			processing_objects -= src
+		return
+	else
+		processing_objects |= src
+
+/obj/item/clothing/suit/hooded/chaplain_hoodie/missionary_robe/process()
+	if(!linked_staff)	//if we don't have a linked staff, the rest of this is useless
+		return
+
+	var/mob/living/carbon/human/H = loc
+
+	if(!istype(H))		//if we somehow try to process while not on a human, remove ourselves from processing and return
+		processing_objects -= src
+		return
+
+	if(linked_staff.faith >= 100)	//if the linked staff is fully recharged, do nothing
+		return
+
+	if(!linked_staff in range(3, get_turf(src)))		//staff won't charge at range (to prevent it from being handed off / stolen and used)
+		if(prob(10))	//10% chance per process should avoid being too spammy, can tweak if it ends up still being too frequent.
+			to_chat(H, "<span class='warning'>Your staff is unable to charge at this range. Get closer!</span>")
+		return
+
+	linked_staff.faith += 5

```

Comment:
you could do away with all the returns if faith is >= 100 if this just followed proper caps with the proper procs lie `min`